homework-3/cycles/script.js

```diff
@@ -0,0 +1,71 @@
+// 1) На основе строки “i am in the easycode” сделать новую строку где первые буквы каждого слова будут в верхнем регистре. Использовать for или while.
+
+var string = "i am in the easycode";
+var space = "";
+
+for (var i = 0; i < string.length; i++) {
+  space += (string[i - 1] == ' ') ? string[i].toUpperCase() : string[i];
```

Comment:
It's good but you forgot about first letter. It is still not uppercase
`for (var i = 0; i < string.length; i++) {`
    `if(i === 0) {`
        `space += string[i].toUpperCase();`
  `  }`
   ` space += (string[i - 1] === ' ')`
     `   ? string[i].toUpperCase()`
      `  : i!== 0 ? string[i] : '';`
`}`

Something like this